packages/camlidl/camlidl.1.07/files/disable-fatal-warn-31.diff

```diff
@@ -0,0 +1,24 @@
+diff -Naur camlidl-1.05/config/Makefile.unix camlidl-1.05-patched/config/Makefile.unix
+--- camlidl-1.05/config/Makefile.unix	2002-04-22 12:50:46.000000000 +0100
++++ camlidl-1.05-patched/config/Makefile.unix	2016-05-04 13:10:23.370704022 +0100
+@@ -37,7 +37,7 @@
+ BINDIR=/usr/local/bin
+ 
+ # The Objective Caml compilers (the defaults below should be OK)
+-OCAMLC=ocamlc -g
++OCAMLC=ocamlc -g -warn-error -31
```

Comment:
This patch is called `disabled-fatal-warn` but enables a fatal warning. Fatal warnings should not be part of any releases

Reply:
oh right, my bad